packages/server/lib/controllers/connection.controller.ts

```diff
@@ -534,6 +534,12 @@ class ConnectionController {
                     errorManager.errRes(res, 'missing_access_token');
                     return;
                 }
+
+                if (!parsedExpiresAt && noExpiration && noExpiration !== true) {
```

Comment:
why the `&& noExpiration`? isn't the last part enough?

Reply:
Thanks, removed with https://github.com/NangoHQ/nango/pull/1827/commits/964a99c105175d9931dc57d9daf042728cade61e